benchmarks/benchmarks/ag_methods.py

```diff
@@ -202,7 +204,78 @@ def time_wrap(self, num_atoms):
         atomgroup with default params.
         """
         self.ag.wrap()
-        
+
+    def time_center_of_mass_default(self, num_atoms):
+        """Benchmark center_of_mass calculation with
+        pbc and unwrap inactive.
+        """
+        self.ag.center_of_mass()
+
+    def time_center_of_mass_pbc(self, num_atoms):
+        """Benchmark center_of_mass calculation with
+        pbc active.
+        """
+        self.ag.center_of_mass(pbc=True)
+
+    def time_center_of_mass_unwrap(self, num_atoms):
+        """Benchmark center_of_mass calculation with
+        unwrap active.
+        """
+        self.ag_unwrap.center_of_mass(unwrap=True, compound='residues')
```

Comment:
These look cool, but some are only going to work from 0.20.0 onwards.  Is there a way to skip benchmarks if they're not valid? (@tylerjereddy )  Some sort of `@requires('0.20.0')` decorator?

Reply:
It is certainly common for imports to be wrapped in try: except: pass to skip the benchmarks that don't have the imports.

Not sure how easy it would be to cook that up for this case though.

Perhaps more promising is that [if setup raises a `NotImplementedError`, the benchmark is marked as skipped](https://asv.readthedocs.io/en/stable/writing_benchmarks.html#setup-and-teardown-functions).